packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1436,7 +1436,9 @@ class _FloatingActionButtonTransitionState extends State<_FloatingActionButtonTr
 /// within the [Scaffold]. The [FloatingActionButton] is connected to a
 /// callback that increments a counter.
 ///
-/// ![The Scaffold has a white background with a blue AppBar at the top. A blue FloatingActionButton is positioned at the bottom right corner of the Scaffold.](https://flutter.github.io/assets-for-api-docs/assets/material/scaffold.png)
+/// ![The Scaffold has a white background with a blue AppBar at the top. A blue
```

Comment:
This will actually be visible to the user since this code sample is a dartpad. Can you revert this so it is not shown twice?

Reply:
I think I can revert it to save you some time revisiting this again. :)

Reply:
Oh when I looked at the diff I realized the image here is redundant, so I just removed it. :)

![image](https://user-images.githubusercontent.com/16964204/236551733-55b96732-2457-455d-9466-883395b7f7c8.png)

Reply:
Oh got it. Thanks!